etc/listproviders/event.upload.asset.options.properties

```diff
@@ -32,15 +32,15 @@ list.name=eventUploadAssetOptions
 # EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_WEBVTT={"id":"attachment_captions_webvtt", "type": "attachment", "flavorType": "text", "flavorSubType": "vtt", "displayOrder":3, "accept": ".vtt"}
-# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.PREVIEW_IMAGE={"id":"attachment_preview_image", "type":"attachment", "flavorType": "presenter","flavorSubType": "search+preview", "displayOrder":4, "accept": "image/*"}
+# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.PREVIEW_IMAGE={"id":"attachment_preview_image", "type":"attachment", "flavorType": "presenter","flavorSubType": "search+preview", "displayOrder":4, "accept": ".bmp,.gif,.jpeg,.jpg,.png,.tif,.tiff"}
 EVENTS.EVENTS.NEW.UPLOAD_ASSET.WORKFLOWDEFID=publish-uploaded-assets
 
 # The video source track upload options are only for new events.
 # Unlike the other assets, multiple source tracks can be uploaded for a single flavor.
 # The MULTIPLE_PARTS example shows how to enable choosing multiple source files for a single flavor. In this case, a fictional "multipart/part+source".
-# EVENTS.EVENTS.NEW.SOURCE.UPLOAD.MULTIPLE_PARTS={"id": "track_parts","type":"track", "flavorType": "multipart","flavorSubType": "part+source", "multiple":true, "displayOrder":11, "accept": "video/*,audio/*"}
-# EVENTS.EVENTS.NEW.SOURCE.UPLOAD.AUDIO_ONLY={"id": "track_audio","type":"track", "flavorType": "presenter-audio", "flavorSubType": "source", "multiple":false, "displayOrder":12, "accept": "audio/*"}
-EVENTS.EVENTS.NEW.SOURCE.UPLOAD.NON_SEGMENTABLE={"id": "track_presenter","type":"track", "flavorType": "presenter","flavorSubType": "source", "multiple":false, "displayOrder":13, "accept": "video/*,audio/*"}
-EVENTS.EVENTS.NEW.SOURCE.UPLOAD.SEGMENTABLE={"id": "track_presentation","type":"track", "flavorType": "presentation","flavorSubType": "source", "multiple":false, "displayOrder":14, "accept": "video/*,audio/*"}
+# EVENTS.EVENTS.NEW.SOURCE.UPLOAD.MULTIPLE_PARTS={"id": "track_parts","type":"track", "flavorType": "multipart","flavorSubType": "part+source", "multiple":true, "displayOrder":11, "accept": ".avi,.flv,.m4v,.mkv,.mov,.mp4,.mpeg,.mpg,.ogv,.webm,.wmv,.flac,.m4a,.mp3,.ogg,.wav,.wma"}
+# EVENTS.EVENTS.NEW.SOURCE.UPLOAD.AUDIO_ONLY={"id": "track_audio","type":"track", "flavorType": "presenter-audio", "flavorSubType": "source", "multiple":false, "displayOrder":12, "accept": ".flac,.m4a,.mp3,.ogg,.wav,.wma"}
+EVENTS.EVENTS.NEW.SOURCE.UPLOAD.NON_SEGMENTABLE={"id": "track_presenter","type":"track", "flavorType": "presenter","flavorSubType": "source", "multiple":false, "displayOrder":13, "accept": ".avi,.flv,.m4v,.mkv,.mov,.mp4,.mpeg,.mpg,.ogv,.webm,.wmv"}
+EVENTS.EVENTS.NEW.SOURCE.UPLOAD.SEGMENTABLE={"id": "track_presentation","type":"track", "flavorType": "presentation","flavorSubType": "source", "multiple":false, "displayOrder":14, "accept": ".flac,.m4a,.mp3,.ogg,.wav,.wma"}
```

Comment:
If I'm not misinterpreting things, these to lines are controlling the two default upload flavors (`presenter` and `presentation`) which should both be video and audio or maybe even just video. But in any case, none of these is pure audio.

Reply:
You are correct. I do not know how I mixed this up. Good catch!

Fixed.